client-v2/src/actions/ArticleFragments.ts

```diff
@@ -375,12 +375,16 @@ const addImageToArticleFragment = (
   uuid: string,
   imageData: ValidationResponse
 ) =>
-  updateArticleFragmentMeta(uuid, {
-    ...getImageMetaFromValidationResponse(imageData),
-    imageReplace: true,
-    imageCutoutReplace: false,
-    imageSlideshowReplace: false
-  });
+  updateArticleFragmentMeta(
+    uuid,
+    {
+      ...getImageMetaFromValidationResponse(imageData),
+      imageReplace: true,
+      imageCutoutReplace: false,
+      imageSlideshowReplace: false
+    },
+    true
```

Comment:
Tiny comment but could name the argument to make sure the purpose of the boolean is clear.